### ISSUE TITLE
Language list on posts fails contrast in light theme

### ISSUE THREAD

**Issue by ChasBelov** [mastodon/mastodon#26948]
### Steps to reproduce the problem

If I am composing a post and choose the language drop-down, the common name of each language fails WCAG 2.0 AA for contrast.

Steps:
1. In a post, click the language code, for example, if you normally interact with Mastodon in English, click EN.

### Expected behaviour

Both the native name and the common name of each language in the list meet WCAG 2.0 AA for contrast.

### Actual behaviour

The native name of each language in the list meets WCAG 2.0 AA for contrast, but the common name does not.

### Detailed description

The class language-dropdown__dropdown__results__item__native-name has the style color: #000 which it inherits from language-dropdown__dropdown__results__item. When displayed over a white background, it produces a contrast of 21:1, which far exceeds the WCAG 2.0 AA requirement of 4.5:1.

The class language-dropdown__dropdown__results__item__common-name, however, has the style color: #c0cdd9. When displayed over a white background, it produces a contrast of 1.6:1, which fails the WCAG 2.0 AA requirement of 4.5:1.

### Mastodon instance

ohai.social

### Mastodon version

v4.2.0-nightly.2023-09-01

### Browser name and version

Firefox 117.0.1

### Operating system

macOS 13.5.2

**Comment by renchap** [comment]
The composer has been completely redone, this has been fixed
